sql/core/src/test/scala/org/apache/spark/sql/connector/AlterTableTests.scala

```diff
@@ -471,6 +492,27 @@ trait AlterTableTests extends SharedSparkSession {
     }
   }
 
+  test("AlterTable: update column position") {
```

Comment:
I think the new tests should assert schema equality like the other test cases. That checks expected type and comments as well as position.

Reply:
done